maintenance_plan/models/maintenance_equipment.py

```diff
@@ -29,11 +29,13 @@ def _compute_maintenance_plan_count(self):
     def _compute_team_required(self):
         for equipment in self:
             equipment.maintenance_team_required = len(
-                equipment.maintenance_plan_ids) >= 1
+                equipment.maintenance_plan_ids.filtered(
```

Comment:
Is it more efficient than casting it with `bool()` ?

Reply:
When checking bool it is checking if len > 0, so it should be the same